packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/InstanceManager.java

```diff
@@ -18,13 +18,24 @@ void addInstance(Object instance, long instanceId) {
     instanceIdsToInstances.append(instanceId, instance);
   }
 
-  /** Remove the instance from the manager. */
-  void removeInstance(long instanceId) {
+  /** Remove the instance with instanceId from the manager. */
```

Comment:
Please describe the return value in the comment.

Reply:
done